code/modules/mob/mob_movement.dm

```diff
@@ -339,7 +339,8 @@
 		//Wheelchair pushing goes here for now.
 		//TODO: Fuck wheelchairs.
 		if(istype(mob.pulledby, /obj/structure/bed/stool/chair/office/wheelchair) || istype(mob.pulledby, /obj/structure/janitorialcart))
-			move_delay += 1
+			if(istype(mob.pulledby, /obj/structure/bed/stool/chair/office/wheelchair))
+				move_delay += 1
```

Comment:
This is really ugly. Remove all of this, make it use `slowdown` instead and add a slowdown of 0 to janicarts and 1 to wheelchairs.

Reply:
addressed.